js/index.js

```diff
@@ -0,0 +1,147 @@
+const btnMenu = document.querySelector('.navbar-toggler');
+const iconBtn = document.querySelector('#icon-btn');
+
+if (btnMenu) {
+  btnMenu.addEventListener('click', () => {
+    iconBtn.classList.toggle('fa-xmark');
+  });
+}
+const featureCooks = document.querySelector('#features-cook');
+const featuresData = [
+  {
+    name: 'Massimo Bottura',
+    job: 'Chef Patron, Osteria Francescana (Modena, Italy)',
+    description: `Massimo Bottura is an Italian chef who owns and runs the three-Michelin-starred Osteria Francescana, located in Modena, Italy. Bottura's dishes are inspired by traditional Italian cuisine, but he adds his own creative twists and modern techniques to make them truly unique. He is known for his commitment to sustainability and his efforts to reduce food waste, as well as his philanthropic work through his non-profit organization, Food for Soul.
+    `,
+    image: 'images/Massimo Bottura.webp',
+  },
+  {
+    name: 'Dominique Crenn',
+    job: 'Chef/Owner, Atelier Crenn (San Francisco, USA)',
+    description: `
+    Dominique Crenn is a French-American chef and restaurateur, and the first female chef in the US to earn three Michelin stars. She is the owner of Atelier Crenn, a highly acclaimed restaurant in San Francisco, where she creates innovative, multi-course tasting menus that reflect her personal experiences and emotions. Crenn is known for her commitment to sustainability and her efforts to promote diversity and inclusion in the culinary industry.`,
+    image: 'images/Dominique Crenn.jpg',
+  },
```

Comment:
- [ ] Please fix the size of the `feature cooks` images if you check your design the size of `cooks` is very small and it does not match with Bahance design so kindly fix it :+1: check below your design and Behance design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-04-20 19-52-37](https://user-images.githubusercontent.com/45226848/233410132-2533c471-3391-4bd8-a742-328f72776426.png)

**Bahance Design**
![Screenshot from 2023-04-20 19-52-23](https://user-images.githubusercontent.com/45226848/233410118-86693db6-5a07-4609-8201-d29908f73d2d.png)

- [ ] Please reduce the length of the paragraph then it looks good and professional :+1: